### ISSUE TITLE
Pixel polishing for Diffraction screen

### ISSUE THREAD

**Issue by arouinfar** [phetsims/wave-interference#377]
@ariel-phet and I discussed some minor pixel polishing for the Diffraction screen. There is some Illustrator work to do, so I'll self assign for now.

**Comment by arouinfar** [comment]
@samreid here are the requested pixel polishes. Please let me know if you would like to discuss any of the changes over Zoom.

### Wavelength panel
- [ ] Top-align with scale bar
- [ ] Left-align with laser
- [ ] Format like Wavelength NumberControl in Bending Light -- Readout between tweakers above track
- [ ] Increase width of slider track 50%
- [ ] Match height of slider track to FrequencySlider on earlier screens
- [ ] Resize panel to fit updated contents 

### Aperture control panel
- [ ] Bottom-align with `ResetAllButton`
- [ ] Increase panel height 10%
- [ ] Increase thumb height 10%
- [ ] Reduce width of Waving Girl panel to match other scenes with 2 NumberControls

### Misc
- [ ] Match position of `ResetAllButton` to first 3 screens
- [ ] Reduce height of laser by 20%
- [ ] Match panel color/strokes to first 3 screens


Here's a mockup of the changes, and another showing the changes overlaid on the original. (Mockups don't show updated `ResetAllButton` position.)
<img width="1024" alt="Screen Shot 2019-04-30 at 9 52 56 PM" src="https://user-images.githubusercontent.com/8419308/57005141-7e4ec700-6b92-11e9-8013-dae1e3ac8fa4.png">
<img width="1025" alt="Screen Shot 2019-04-30 at 9 53 57 PM" src="https://user-images.githubusercontent.com/8419308/57005142-7e4ec700-6b92-11e9-9fea-7b186155a5b7.png">

**Comment by samreid** [comment]
> Bottom-align with ResetAllButton

This was already in place

**Comment by samreid** [comment]
> Reduce width of Waving Girl panel to match other scenes with 2 NumberControls

With the English strings, the discrepancy between the widths is caused by the lengths of the tick labels.  Changing the minimum angle from "0" to "0.00" nearly completely unifies these sizes.  Brainstorming ways to proceed:

a) Change the minimum angle from "0" to "0.00".  Doesn't exactly match, but much closer (off by a few pixels).  Easy to implement, but not very robust--changes in labels or text size could throw things off.
b) Add another layer in the containers for these panels, then match sizes with AlignGroup.  Difficult to implement (maybe 30-60 minutes if AlignGroup works for this), more robust.  But increases the complexity of the code significantly.
c) Leave the panels to have different widths.  Easy to implement and doesn't increase code complexity.

@arouinfar I wanted to check in with you about these options before spending much time on that part.  What do you recommend?

**Comment by arouinfar** [comment]
>>Bottom-align with ResetAllButton

>This was already in place

This was requested in case the alignment shifted when the ResetAllButton's position was moved to match the first three screens.

> With the English strings, the discrepancy between the widths is caused by the lengths of the tick labels. Changing the minimum angle from "0" to "0.00" nearly completely unifies these sizes. Brainstorming ways to proceed:

Ah, that makes sense. Let's leave it be, then.

**Comment by samreid** [comment]
I've moved the pixel polishing requests for the wavelength slider to a separate issue, since https://github.com/phetsims/wave-interference/issues/394 must be completed first.  @arouinfar can you please review the other parts of this issue and see if there's anything left to do?

**Comment by arouinfar** [comment]
Thanks @samreid, looks great! Since the remaining work is being tracked in #394, I'll go ahead and close.
